chapters/CRUD-Operations.md

```diff
@@ -617,10 +607,10 @@ This algorithm takes in a `capabilityId` and returns a `rootCapability` object.
 1. Set `rootCapability` to an empty object.
 1. Set `components` to the result of `capabilityId.split(":")`.
 1. Validate `components`:
-    1. Assert length of `components` is 4.
-    1. `components[0] == urn`.
-    1. `components[1] == zcap`.
-    1. `components[2] == root`.
+   1. Assert length of `components` is 4.
+   1. `components[0] == urn`.
+   1. `components[1] == zcap`.
+   1. `components[2] == root`.
 1. Set `uriEncodedId` to `components[3]`.
```

Comment:
should there be an assertion of what components[3] is before we assign the value to uriEncoded?

Reply:
I don't think so. Because it is just some identifier, can be any URI encoded DID btc1. We should check that it is a did:btc1 when decoding it I think.